classifiers/reference_quality/special_character_classifier/code_snippet_common.md

```diff
@@ -0,0 +1,44 @@
+```python
+import unicodedata
+from typing import List
+
+def special_character_classifier(text: str, allowed_ranges: list = None) -> bool:
```

Comment:
typing is wrong i think. usually we use something list `List[Tuple[int,int]]`

Reply:
For extractors, we have `List[Tuple[str, int]` yes, but this brick is a classifier and just returns a boolean. It might make sense to return a string however for consistency. (The refinery code returns a boolean as well, which seems to work for the labeling functions though? Not sure if this could cause trouble)

Reply:
the typing of allowed_ranges variable not the return type was ment.

For refinery bool is fine if it's a generator